v1.1.yml

```diff
@@ -1259,28 +1305,36 @@ definitions:
         x-nullable: true
       student:
         type: string
+        x-validation: true
       type:
         type: string
-        x-nullable: true
 
   Teacher:
     type: object
     properties:
       id:
         type: string
+        x-validation: true
       created:
         type: string
         format: datetime
+        x-validation: true
+      credentials:
+        $ref: "#/definitions/Credentials"
         x-nullable: true
       district:
         type: string
+        x-validation: true
       email:
         type: string
         x-nullable: true
+      google_apps:
+        $ref: "#/definitions/GoogleApps"
+        x-nullable: true
       last_modified:
         type: string
         format: datetime
-        x-nullable: true
+        x-validation: true
       name:
         $ref: "#/definitions/Name"
```

Comment:
This might be a problem because we reuse the definition, but for Student, Teacher and School Admin first and last name are required. For district admin they are not